### PR TITLE
[Auth0 FLAC 2/3] DSS Ops: Flac Support

### PR DESCRIPTION
**Pull request by amarjandu**
adds functionality to add, list,remove keys from the flac dynamodb table

### CODE REVIEW

File: dss/operations/flac.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+checkout = dispatch.target("flac", help=__doc__)
+
+
+@checkout.action("inspect",
+                 arguments={"--keys": dict(nargs="+", help="Keys to inspect in DynamoDB", required=True)})
+class Inspect(FlacHandler):
+    def process_keys(self):
+        key_status = []  # used to house the status of they keys we will inspect
+        """Remove keys from the checkout bucket."""
```
Comment (chmreid) [inline dss/operations/flac.py]: Need to update this docstring!

File: dss/operations/flac.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            uuid, version = self._parse_key(_key)
+            temp_status = {"fqid": _key}
+            try:
+                flac_attributes = ddb.get_item(table=self.flac_lookup_table_name,hash_key=uuid)
+            except ddb.DynamoDBItemNotFound:
+                # nothing was found within the database
+                pass
+            else:
+                temp_status.update(flac_attributes)
+            key_status.append(temp_status)
```
Comment (chmreid) [inline dss/operations/flac.py:59]: Should print or log or something

**Comment by chmreid** [comment]
Looks like this isn't quite ready for review :P

**Comment by amarjandu** [comment]
> Looks like this isn't quite ready for review :P

wowowow calling me out like that? i just wanted to click all the side bar buttons, i still gotta include the tests PR. :p

File: dss/dynamodb/__init__.py [hunk 3]
```diff
@@ -65,9 +71,7 @@ def get_item(*, table: str, hash_key: str, sort_key: Optional[str] = None) -> ty
         raise
     if item is None:
         raise DynamoDBItemNotFound(f'Query failed to fetch item from database: {query}')
-    for k, v in item.items():  # strips out ddb typing info
-        return_value[k] = [*v.values()][0]
-    return return_value
+    return _format_ddb_response(item)
```
Comment (chmreid) [inline dss/dynamodb/__init__.py:79]: clean

File: dss/dynamodb/__init__.py [hunk 4]
```diff
@@ -143,3 +147,31 @@ def delete_item(*, table: str, hash_key: str, sort_key: Optional[str] = None):
     query = {'TableName': table,
              'Key': _format_item(hash_key=hash_key, sort_key=sort_key, value=None)}
     db.delete_item(**query)
+
+
+def update_item(*, table: str, hash_key: str, sort_key: Optional[str] = None, update_expression: Optional[str],
```
Comment (chmreid) [inline dss/dynamodb/__init__.py:157]: I think this was already brought up in a prior code review comment, but this should have a test (will follow up and dismiss this comment if there is one in a later PR)

Comment (chmreid) [reply dss/dynamodb/__init__.py:157]: Later PR added a test of another function that uses `update_item()` function, so there is indirect testing.

Can leave writing a test for this function directly as a future TODO item.

**Review by chmreid — APPROVED** [review]
LGTM

**Pull request merged**